apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx

```diff
@@ -58,20 +64,23 @@ export const ApplicationDetailsFooter = ({
             className="shrink-0 text-accent-primary md:hidden [&_path]:fill-current"
             size={24}
           /> */}
-          {(isMyAppsTab || isMyApp) && (
-            <Tooltip tooltip={isMyApp ? t('Delete') : t('Remove')}>
-              <button
-                onClick={() => (isMyApp ? onDelete(entity) : onRemove(entity))}
-                className="group flex size-[34px] items-center justify-center rounded text-secondary hover:bg-accent-primary-alpha hover:text-accent-primary"
-                data-qa="application-edit"
-              >
-                <IconTrashX
-                  size={24}
-                  className="shrink-0 group-hover:text-accent-primary"
-                />
-              </button>
-            </Tooltip>
-          )}
+          {(isMyAppsTab || isMyApp) &&
+            installedModelIds.has(entity.reference) && (
```

Comment:
```suggestion
          {(isMyApp ||
            installedModelIds.has(entity.reference)) && (
```

Reply:
If i remove condition, then the installed model (if it is not my application but an installed model) will not have a remove button in my workspace when the card is open

Reply:
thats why Ilya suggested to replace `&&` with `||`